tests/features/bootstrap/FeatureContext.php

```diff
@@ -75,4 +75,19 @@ public function iCreateColorTerm() {
     $element->findButton('Save')->click();
   }
 
+  /**
+   * Creates a menu item with specified name in the specified menu.
+   *
+   * @Given /^I create an item "([^"]*)" in the "([^"]*)" menu$/
+   */
+  public function iCreateItemInTheMenu($menu_item, $menu_name) {
+    $path = '/admin/structure/menu/manage/' . $menu_name . '/add';
+    $this->getSession()->visit($this->locatePath($path));
+    $element = $this->getSession()->getPage();
+    $element->fillField('Menu link title', $menu_item);
+    $element->fillField('Link', 'http://example.com');
+    $element->checkField("Show as expanded");
+    $element->findButton('Save')->click();
```

Comment:
Why do we need this method? Can we just use already existing steps in behat like you do in `Create menu item for second level`?

The reason why we should avoid custom commands, because they will require maintenance in future. Less custom code - better. We should use already build in steps.